src/components/generic/OptionsList/OptionsList.jsx

```diff
@@ -6,6 +6,82 @@ import Icon from '../Icon/Icon';
 import webexComponentClasses from '../../helpers';
 import {useRef, useAutoFocus} from '../../hooks';
 
+/**
+ * Option Component
+ *
+ * @param {string} className  Custom CSS class to apply
+ * @param {boolean} focused  Focused option
+ * @param {Function} onKeyDown  A function that will be triggered when a key is pressed
+ * @param {Function} onMouseEnter  A function that will be triggered on mouse hover
+ * @param {Function} onSelect A function which will be triggered on option selection
+ * @param {object} option  An option from options list
+ * @param {Function} sc CSS subclass function
+ * @param {boolean} selected  Selected option label
```

Comment:
this can be a boolean

Reply:
will not be ok here: {selected && <Icon className = {classNames (sc ('check')...